platform/ios/resources/zh-Hans.lproj/Localizable.strings

```diff
@@ -0,0 +1,84 @@
+/* Accessibility hint */
+"ANNOTATION_A11Y_HINT" = "显示信息";
+
+/* No comment provided by engineer. */
+"API_CLIENT_400_DESC" = "The session data task failed. Original request was: %@";
+
+/* No comment provided by engineer. */
+"API_CLIENT_400_REASON" = "The status code was %ld";
+
+/* No comment provided by engineer. */
+"CANCEL" = "取消";
+
+/* Accessibility hint */
+"COMPASS_A11Y_HINT" = "旋转地图使正北朝上";
+
+/* Accessibility label */
+"COMPASS_A11Y_LABEL" = "指南针";
+
+/* Compass abbreviation for north */
+"COMPASS_NORTH" = "N";
+
+/* Copyright notice in attribution sheet */
+"COPY_MAPBOX" = "© Mapbox";
+
+/* Copyright notice in attribution sheet */
+"COPY_OSM" = "© OpenStreetMap";
+
+/* Instructions in Interface Builder designable; {key}, {plist file name} */
+"DESIGNABLE" = "在%2$@中将你的access token设为%1$@可在这里显示Mapbox上的地图\n\n更多说明请见";
+
+/* Setup documentation URL display string; keep as short as possible */
+"FIRST_STEPS_URL" = "mapbox.com/help/first-steps-ios-sdk";
+
+/* Accessibility hint */
+"INFO_A11Y_HINT" = "显示致谢，用户反馈及更多";
+
+/* Accessibility label */
+"INFO_A11Y_LABEL" = "关于这个地图";
+
+/* Accessibility label */
+"LOGO_A11Y_LABEL" = "Mapbox";
+
+/* Accessibility label */
+"MAP_A11Y_LABEL" = "地图";
+
+/* Map accessibility value */
+"MAP_A11Y_VALUE" = "Zoom %1$dx\n%2$ld annotation(s) visible";
```

Comment:
After the user zooms the map, VoiceOver says, for example:

> Zoom 2x
> 12 annotations visible

`%1d` is a placeholder for the zoom scale (but `x` is pronounced literally in English). `\n` is a newline. `%2$ld` is a placeholder for the number of visible annotations.

If “annotations” doesn’t translate well, you can try something like “pins”, “markers”, or “map items”.